### DISCUSSION TITLE
Offseason General Chat

### DISCUSSION THREAD

**Discussion by jkleiber** [team401/2024-Robot-Code#195] [General]
@aidnem @minhnguyenbhs @linglejack06 I'm creating this chat so we can discuss anything that comes up during the offseason/summer outside of shop hours. 

This is a space mainly for y'all to coordinate work and decide on the priorities that you want to execute. For specific topics, we can create separate GitHub discussions or tickets depending on the scope.

This doesn't replace the use of the ticketing system or discussions in PRs, but I want to make sure y'all are aware that we have a forum for general questions and discussions in the event of unexpected interruptions to the shop calendar, or expected breaks in meetings (summer). In general, I am available all year round for online discussion, reviewing your PRs, and working on building our codebase up regardless of if shop is open or not. Our goal for the near future is to get to the point where we can work just as well regardless of if we have the robot or not.

cc: @PChild

**Comment by linglejack06** [comment]
@jkleiber are we doing anything important in shop today? I just got finished with AP exam but I can still come to shop I just might not be able to make it until 6:45. If we can use robot though I'd like to test out demo mode

**Reply by jkleiber** [reply]
@linglejack06 I won't be there this week but I imagine that you guys could test out demo mode and the shooter monitoring today.

**Reply by linglejack06** [reply]
@aidnem are you coming to shop tonight?

**Reply by aidnem** [reply]
@jkleiber We got the new radios (6ghz) and spent the whole shop getting them configured and working. It gets concerningly hot (at least for my liking) but apparently that's fine and the radio seems to work (got communication light green in driver station). Then Wednesday was the last shop of the year with kickball, so no testing got done. Maybe some optional summer shop will see more progress.

**Reply by jkleiber** [reply]
Thanks for the update, and have a good rest of the school year!

**Comment by aidnem** [comment]
@jkleiber I'm not sure if you've seen this, but Preston and Oliver have both asked me about it. Do you think it's a valuable use of our time (maybe after offseason comps we could play with it)
[Realistic Physics within Sim](https://www.chiefdelphi.com/t/6328s-advance-swerve-drive-example-with-more-enhanced-physics-simulation-collision-detection-and-more/470626)

**Reply by jkleiber** [reply]
@aidnem yeah we should definitely explore this after offseason competitions are over. Could you create a ticket for this (maybe it belongs in coppercore as a potential sim module)
